app/src/page/file/view.js

```diff
@@ -596,13 +591,17 @@ class FilePage extends React.PureComponent {
                           style={[filePageStyle.actionButton, filePageStyle.bellButton]}
                           uri={fullChannelUri}
                           name={channelName} />
+                        <TouchableOpacity style={filePageStyle.descriptionToggle}
+                          onPress={() => this.setState({ showDescription: !this.state.showDescription })}>
+                          <Icon name={this.state.showDescription ? "caret-up" : "caret-down"} size={24} />
+                        </TouchableOpacity>
                       </View>
                     </View>
                   }
 
-                  {description && description.length > 0 && <View style={filePageStyle.divider} />}
-
-                  {description && <Text style={filePageStyle.description} selectable={true}>{this.linkify(description)}</Text>}
+                  {(this.state.showDescription && description && description.length > 0) && <View style={filePageStyle.divider} />}
```

Comment:
You could use `React.Fragment` to wrap the line and description. Then you don't need two checks that a description exists.